ssh/ssh.go

```diff
@@ -204,7 +205,7 @@ func scpPut(src, dest string, progress func(float64), session *ssh.Session) erro
 // TODO(benesch): Make progress handling for directories less confusing. The
 // SCP protocol makes this challenging, as it does not send the total size of
 // all files.
-func scpGet(src, dest string, progress func(float64), session *ssh.Session) error {
+func ScpGet(src, dest string, progress func(float64), session *ssh.Session) error {
```

Comment:
Go camel-casing nit: these are traditionally spelled `SCPGet` and `SCPPut` when capitalized.